api/src/test/java/org/openmrs/module/shr/odd/api/impl/test/OnDemandDocumentServiceImplTest.java

```diff
@@ -69,8 +69,6 @@ public void beforeTest() throws Exception
 		Context.getAdministrationService().setGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_FALSE_CONCEPT, "1066");
         Context.getAdministrationService().setGlobalProperty("shr-cdahandler.cacheMappedConcepts", "false");
 		Context.getAdministrationService().saveGlobalProperty(saveDir);
-		GlobalProperty disableValidation = new GlobalProperty(OpenmrsConstants.GP_DISABLE_VALIDATION, "true");
-		Context.getAdministrationService().saveGlobalProperty(disableValidation);
```

Comment:
I see, it would be useful to keep this if possible as it provides a massive performance benefit. I don't think strict validation is that beneficial when the system is being used as an SHR. What would the implications of making the minimum version 2.0? Should these modules just work or will they need some refactoring?

I'm fine to drop this if it's difficult to migrate to 2.0.